code/modules/events/supermatter_surge.dm

```diff
@@ -13,7 +13,7 @@
 	var/power = 2000
 
 /datum/round_event/supermatter_surge/setup()
-	power = rand(400,8000)
+	power = rand(1000,100000)
```

Comment:
Thats a very wide range, this will lead to inconsistent gameplay.

Reply:
Yes and the severity is going to be stated in the announcement so it's not like it's secret.